packages/material-ui/src/Tab/Tab.js

```diff
@@ -54,13 +53,15 @@ const TabRoot = styled(ButtonBase, {
   overflow: 'hidden',
   whiteSpace: 'normal',
   textAlign: 'center',
+  flexDirection: 'column',
+  lineHeight: 1.25,
   /* Styles applied to the root element if both `icon` and `label` are provided. */
   ...(styleProps.icon &&
     styleProps.label && {
       minHeight: 72,
       paddingTop: 9,
       paddingBottom: 9,
-      [`& .${tabClasses.wrapper} > *:first-child`]: {
+      [`& > *:first-child`]: {
         marginBottom: 6,
```

Comment:
@oliviertassinari @eps1lon @mnajdova any concern if we use `gap: 6` instead of this. It will make customization easier. For Tabs, I think some of the use cases is to make icon & label as row. With current css, this is what I have to do.

```js
<Tab
  icon={Svg}
  label="Label"
  sx={{
    flexDirection: 'row',
    '& > *:first-child': {
      marginBottom: 0,
      marginRight: 6,
    }
  }}
/>
```

But with gap,
```js
<Tab
  icon={Svg}
  label="Label"
  sx={{
    flexDirection: 'row',
  }}
/>
```

Reply:
How does browser support of `gap` look like?

Reply:
@eps1lon Good point. Overall 75% but most impact is on safari mobile (11% not supported). I think we can wait.

<img width="1379" alt="Screen Shot 2564-06-24 at 16 08 07" src="https://user-images.githubusercontent.com/18292247/123235788-653dc880-d506-11eb-8730-aa63457004ef.png">

Reply:
Yeah, safari mobile is pretty important. That's like 30% of mobile usage or something in that ballpark.

Reply:
The mention of first-child makes me think of #24894 but it's not directly related. 11% of the market indeed seems too early.

Reply:
> 11% of the market

There is no such thing as "the market". I'm mentioning this since this leads to harmful decisions.

Reply:
Right, 11% of the active user agents worldwide.

Reply:
> Right, 11% of the active user agents worldwide.

Which is a meaningless number. This would affect ~86% of iPhone users